webrtc.html

```diff
@@ -1297,6 +1299,25 @@
               </li>
 
               <li>
+                <p>If <var>candidate</var> is <code>null</code>, then run the
+                following steps:</p>
+                <ol>
+                  <li>
+                    <p>For each media section in the last successfully applied
+                    remote description, perform the processing for an
+                    end-of-candidates indication for said media section as
+                    defined in [[TRICKLE-ICE]], unless an ICE restart occurred
+                    for said media section since setting the last remote
+                    description.</p>
```

Comment:
This caveat exists to avoid the race condition where one peer sets a local offer triggering an ICE restart at the same time as another peer signals end-of-candidates. In this situation, an "end-of-candidates" received before the remote answer refers to the previous "generation" of ICE candidates, so it should be ignored.